all/pom.xml

```diff
@@ -428,7 +428,7 @@
                             <createSourcesJar>true</createSourcesJar>
                             <promoteTransitiveDependencies>false</promoteTransitiveDependencies>
                             <keepDependenciesWithProvidedScope>false</keepDependenciesWithProvidedScope>
-                            <createDependencyReducedPom>false</createDependencyReducedPom>
+                            <createDependencyReducedPom>true</createDependencyReducedPom>
```

Comment:
The switch is open ,  then can create createDependencyReducedPom .xml files，so please add this file in .gitignore